### ISSUE TITLE
Should we worry about filtering the events at this stage? I can add something to the event page so it can be either part of the score or not?

### ISSUE THREAD

**Issue by hellaeon** [hellaeon/epl_fantasy#4]

**Comment by johnnysammon** [comment]
I think this is definitely good, but just want a bit more time looking at the list to narrow down the stats to meaningful ones. So may stats! 

Maybe a check box on the stats page, and then after refining that we can totally remove it from import?

**Comment by hellaeon** [comment]
yeah I was thinking something like that - on the events selector page have a checkbox. Think of it like a feature if you want to bring it in later....

Date: Thu, 23 Jul 2015 19:05:51 -0700

**Comment by johnnysammon** [comment]
Yeah deffo whack checkboxes on the events selector page. I'm thinking now the best way is to uncheck everything by default, and start adding stats by importance.
Goal, Assist etc
